README.md

```diff
@@ -1,67 +1,74 @@
-# Overview
-
-A simple and efficient blockchain ledger.
-
-## About
+# Hyperledger Iroha
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 ![Rust](https://github.com/hyperledger/iroha/workflows/Rust/badge.svg?branch=iroha2-dev)
 [![codecov](https://codecov.io/gh/hyperledger/iroha/branch/iroha2-dev/graph/badge.svg)](https://codecov.io/gh/hyperledger/iroha)
 
+Iroha is a simple and efficient blockchain ledger based on the  distributed ledger technology (DLT). Its design principles are inspired by the Japanese Kaizen principle of eliminating excesses (muri).
 
-Iroha is a distributed ledger technology (DLT). Its design principles are inspired by the Japanese Kaizen principle — eliminate excesses (muri). Iroha can help you manage your accounts, assets, on-chain data storage with efficient smartcontracts, while being Byzantine- and crash-fault tolerant, being able to withstand a fault rate of 33%.
+Iroha can help you manage your accounts, assets, on-chain data storage with efficient smart contracts, while being Byzantine- and crash-fault tolerant and being able to withstand a fault rate of 33%.
```

Comment:
```suggestion
Iroha can help you manage your accounts, assets, on-chain data storage with efficient smart contracts, while being Byzantine- and crash-fault tolerant with a fault rate of 33%.
```

Reply:
In retrospect we can't explain this without also making it sound like Iroha crashes at a rate of 33% (which is mostly unacceptable).